bears/js/JSHintBear.py

```diff
@@ -254,7 +254,7 @@ def generate_config(filename, file,
         :param environment_phantom:
             This option defines globals available when your core is running
             inside of the PhantomJS runtime environment.
-        :param maxerr:
+        :param max_errors:
```

Comment:
can we rather not expose this and pass something relatively close to infinity instead? :P